### DISCUSSION TITLE
Human Body Types

### DISCUSSION THREAD

**Discussion by moon47usaco** [willwulfken/MidJourney-Styles-and-Keywords-Reference#38] [Ideas and Style Submissions]
I currently want to generate a fighting kung-fu style character but controlling the muscle mass is difficult.

I get either abnormally string or really weak looking when using various weights for big muscles, normal muscles, etc. I am looking for a lean muscle type like a monk that has strength but muscles are not stringy or bulging.

This could of course be broadened to both male and female types,
Skinny to obese,
Frail to super human,
Fit to unhealthy... etc, etc...

**Comment by willwulfken** [comment]
I've added this to my to do list
